File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/ModelContextWindow.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class ModelContextWindow(
  val input: Long,
  val output: Long,
  val context: ContextParams? = null,
)

File: agent/bindings/csharp/ModelContextWindow.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class ModelContextWindow
  {
    [JsonProperty(PropertyName = "input")]
    public int Input { get; set; }
    [JsonProperty(PropertyName = "output")]
    public int Output { get; set; }
    [JsonProperty(PropertyName = "context")]
    public ContextParams Context { get; set; }
  }
}
